.github/dependabot.yml

```
@@ -0,0 +1,17 @@
+version: 2
+
+updates:
+
+  # Maintain dependencies for GitHub Actions
+- package-ecosystem: github-actions
+  directory: /
+  schedule:
+    interval: weekly
+
+  # Maintain dependencies for python
+- package-ecosystem: pip
+  directory: /
+  schedule:
+    interval: daily
```

```suggestion
    interval: weekly
```
Judging from RSS feeds of packages we depend on, a daily interval could end up leading to a few dependabot PRs per week, so I'm wondering if weekly or monthly would be preferable.  Daily might be a good way to start things out to make sure it's working though.

---

I did not originally consider either of these points... but they're both good, so let's go with weekly for now 😁 
```suggestion
    interval: daily  # TODO s/daily/weekly once dependabot has settled
```